elyra/util/cos.py

```diff
@@ -55,9 +55,10 @@ def __initialize_object_store(self):
         try:
             if not self.client.bucket_exists(self.bucket):
                 self.client.make_bucket(self.bucket)
-        except BucketAlreadyOwnedByYou as ex:
+        except BucketAlreadyOwnedByYou:
```

Comment:
This code doesn't make sense to me in the first place.  If we're checking for the bucket's existence (assuming the existence check works), why would we get EITHER of the exceptions `BucketAlreadyOwnedByYou` and `BucketAlreadyExists`??

In my opinion, and given the existence check can't be trusted in the first place, we should call `make_bucket` unconditionally and ignore the two Bucket exceptions.